apps/admin/dev/base/keda-identity.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: keda
+  namespace: admin
+spec:
+  type: 0
+  resourceID: "/subscriptions/74dacd4f-a248-45bb-a2f0-af700dc4cf68/resourceGroups/managed-identities-stg-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/keda-stg-mi"
+  clientID: "8469c7cb-7be4-46c7-a76b-74674726520e"
```

Comment:
this should be in staging and then just included in dev, don't duplicate values across dev and staging